### PR TITLE
Issue #506 - Stick footer to the bottom of the window

### LINKED ISSUE #506: Stick footer to the bottom of the window
Our current footer floats up and down with the content of the page. I'm pretty sure it used to stick to the bottom (see #25). In any event, it does not now and it would be great if it did.

### PR DESCRIPTION
**Pull request by storrisi**
Fixes #506

### CODE REVIEW

File: src/styles/_Footer.scss [hunk 1]
```diff
@@ -7,6 +7,8 @@
   flex: none;
   display: flex;
   font-size: 1.1em;
+  position: absolute;
+  bottom: 0;
```
Comment (dabutvin) [inline src/styles/_Footer.scss]: The footer has to be below all content. 
See the results of this on the homepage

image

**Comment by gianpaj** [comment]
Hi @dabutvin, I've fixed this as well.

File: src/styles/_App.scss [hunk 2]
```diff
@@ -9,6 +9,10 @@
   font-weight: 400;
 }
 
+.App-content {
+  flex: 1;
```
Comment (dabutvin) [inline src/styles/_App.scss]: nice

**Comment by dabutvin** [comment]
👍 looks good

**Pull request merged**